### PR TITLE
Paymentrequest stickersheet

### PR DESCRIPTION
**Pull request by mustafa-x**
Created a stickersheet that designers and developer can use in Sketch

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
Hey @mustafa-x 
Here are few nits:
- Add `paymentrequest/stickersheet/index.html` to [`paymentrequest/index.html`](https://googlechrome.github.io/samples/paymentrequest/) as well. 
- `s/tak/take` in `paymentrequest/stickersheet/index.html`
- Replace `paymentrequest/stickersheet/index.html` header with
```
--- 
feature_name: PaymentRequest Stickersheet
chrome_version: 53 
feature_id: 5639348045217792 
--- 
```

**Comment by mustafa-x** [comment]
hey @beaufortfrancois added changes just now :)

**Comment by beaufortfrancois** [comment]
Thanks! Let me know when you're ready for review and want me to merge.

**Comment by mustafa-x** [comment]
Feel free to review. I am waiting on another PR from Web Fundamentals as this links to that. Once that has been merged then we are good to go.

File: paymentrequest/index.html [hunk 1]
```diff
@@ -39,4 +39,7 @@
   <li><a href="https://emerald-eon.appspot.com">Server side</a> - sample that
     demonstrates server-side calculation of shipping price and processing
     transactions through a payment service provider.</li>
+
+  <li><a href="https://googlechrome.github.io/samples/paymentrequest/stickersheet/index.html">Stickersheet</a>  -
```
Comment (beaufortfrancois) [inline paymentrequest/index.html]: You can simply use `/stickersheet/` there instead of the full URL.

**Review by beaufortfrancois — CHANGES_REQUESTED** [review]
LGTM but nit.

**Comment by beaufortfrancois** [comment]
Here it is live!
https://googlechrome.github.io/samples/paymentrequest/
https://googlechrome.github.io/samples/paymentrequest/stickersheet/

**Pull request merged**